api/routes/routes.js

```diff
@@ -1,32 +1,28 @@
-const
-  express = require('express');
-  bodyParser = require('body-parser')
+'use strict';
 
-// create application/json parser - used for /post method only
-app.use(bodyParser.urlencoded({
-  extended: true
-}));
-app.use(bodyParser.json());
```

Comment:
I left the bodyParser initialisation in because I thought it was a little tricky to find info on for someone who hasn't built an API with JS/express before. Since the associates will add an endpoint that adds a pet to the database, I thought I'd start them off with this part. Thoughts?

Reply:
Ah all good, I see it's been moved to `server.js`

Reply:
yeah, after making it so that `app` wasn't accidentally made global I needed to move these into where `app` is defined so we have a reference to it.